bring/src/main/java/com/bobocode/petros/bring/context/AnnotationConfigApplicationContext.java

```diff
@@ -74,11 +81,18 @@ void registerBeanDefinitions(final String packageName) {
         var reflections = new Reflections(packageName, Scanners.SubTypes.filterResultsBy(s -> true));
         var allClasses = reflections.getSubTypesOf(Object.class);
         Objects.requireNonNull(this.configurationBeanDefinitionScanner, "Configuration scanner must be initialized!");
-        var configurationBeanDefinitions = this.configurationBeanDefinitionScanner.scan(allClasses);
-
+        var beanDefinitions = getBeanDefinitions(allClasses);
```

Comment:
Hello, @crezione1 
Could you please pull the latest changes from master and adapt your code to it? Thanks in advance.